### DISCUSSION TITLE
Inconsistent publishing of CyloneDX Documents for buildpack layers

### DISCUSSION THREAD

**Discussion by xyloman** [paketo-buildpacks/feedback#52] [Ideas]
In order to attribute which buildpack layer contributed a vulnerable package, the cnb-sboms layer should include CylconeDX documents as well as Syft to allow users to use either Grype or Trivy to scan the SBOM. Some layers (e.g. paketo-buildpacks_ca-certificates, paketo-buildpacks_spring-boot). Some layers do include a CDX document for instance `paketo-buildpacks_bellsoft-liberica`.

**Comment by dmikusa** [comment]
For libpak based buildpacks, there is a feature request to [add this](https://github.com/paketo-buildpacks/libpak/issues/154). You can 👍 to vote for that if it is needed.
